src/Swagger.ObjectModel/SwaggerModel.cs

```diff
@@ -29,17 +29,6 @@ public class SwaggerModel
         public string Ref { get; set; }
 
         /// <summary>
-        /// Gets a value indicating whether reference.
```

Comment:
I haven't figured out how to handle references yet. I was just going to start by ignoring them which could potentially result in larger json documents, but at least it would work.